src/utils/utils.js

```diff
@@ -24,6 +24,9 @@ export const CTGROUPS = {
     BR: CTYPES.BR,
 };
 
+export const URL_REGEX = /((?:https?:\/\/)?(?:[a-z0-9-]{1,63}\.){1,2}[a-z]{2,15}(?:\/[^\s]*)?)/gi;
+export const URL_REGEX_WITH_INFOS = /((https?:\/\/)?([a-z0-9-]{1,63}\.){1,2}[a-z]{2,15}(\/[^\s]*)?)/gi;
```

Comment:
I'm just going to trust you on these regexes :eyes:

Reply:
;)